freud/order.pyx

```diff
@@ -649,7 +651,7 @@ cdef class Steinhardt(_PairCompute):
         computed by averaging the :math:`q_{lm}` values (or
         :math:`\overline{q}_{lm}` values if ``average`` is enabled) over all
         particles before computing the rotationally-invariant order
-        parameter."""
+        parameter."""  # noqa W605
```

Comment:
This probably needs to be a raw string instead. This warning is indicating that `\o` is not a valid escape sequence.